spec/features/restaurants/create_update_restaurant_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+feature 'creating and updating restaurants' do
+  context 'as an authorized user' do
+    let(:user) { FactoryGirl.create(:user) }
+    let(:restaurant) { FactoryGirl.create(:restaurant) }
+
+    before :each do
+      sign_in_as user
+    end
+
+    scenario 'I can successfully create a restaurant' do
+      visit new_restaurant_path
+
+      fill_in 'Name', with: restaurant.name
+      fill_in 'Address', with: restaurant.address
+      fill_in 'City', with: restaurant.city
+      fill_in 'Description', with: restaurant.description
+      find('#restaurant_cuisine_id').find(:xpath, 'option[1]').select_option
```

Comment:
From what I've found, this is one of the cleaner options: http://www.farooqyousuf.net/2013/12/how-to-select-option-in-drop-down-using-capybara/